contrib/platform/src/com/sun/jna/platform/win32/Wevtapi.java

```diff
@@ -0,0 +1,702 @@
+/* Copyright (c) 2016 Minoru Sakamoto, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
```

Comment:
It might be better to  to change [Contributing.md](https://github.com/java-native-access/jna/blob/master/www/Contributing.md#copyright-headers-in-files).